File: Frontend/src/app/catalogo/cantidad-producto.ts
import {Oferta} from "./oferta/oferta";
import {PedidoProductor} from "./pedido/pedido-productor/pedido-productor";
import {Producto} from "./producto/producto";

export class CantidadProducto {
  id: number;
  cantidad: number;
  ofertaProductor: any;
  pedidoProductor: any;
  producto: any;
}

File: Backend/MarketPlace_Backend/marketplace/models/cantidad_producto.py
from django.db import models
from .oferta_productor import OfertaProductor
from .producto import producto
from django.core.validators import MinValueValidator


class CantidadProducto(models.Model):
    cantidad = models.IntegerField(validators=[MinValueValidator(0)])
    oferta_productor = models.ForeignKey(OfertaProductor, on_delete=models.CASCADE, blank=False,null=False)
    producto = models.ForeignKey(producto, on_delete=models.CASCADE, blank=False, null=False)